pkg/agent/cniserver/server.go

```diff
@@ -367,12 +357,16 @@ func (s *CNIServer) CmdAdd(ctx context.Context, request *cnipb.CniCmdRequest) (*
 	cniVersion := cniConfig.CNIVersion
 	result := &current.Result{CNIVersion: cniVersion}
 	netNS := s.hostNetNsPath(cniConfig.Netns)
+	isInfraContainer := isInfraContainer(netNS)
 
 	success := false
 	defer func() {
 		// Rollback to delete configurations once ADD is failure.
 		if !success {
 			klog.Warningf("CmdAdd has failed, and try to rollback")
```

Comment:
Revise the log as you do not rollback for workload containers..

Reply:
Done.